### PR TITLE
Fix send contacts

### PR DESCRIPTION
**Pull request by YehudaEi**
There is a problem sending contacts, and this fix will fix it.

(It is very possible that this fault also exists in other types of messages where this function is not static).

### CODE REVIEW

**Comment by adiwajshing** [comment]
@YehudaEi your fix edits an auto-generated file, it's not going to fix any issue

1. if there is a bug in sending contacts, describe it, attach more info -- we can't go by vague descriptions
2. if there is a bug when sending other types of messages -- again, describe the issue

**Comment by YehudaEi** [comment]
@adiwajshing 
Sorry, I just do not speak typescript ...

My problem occurred while trying to run the example that appears in the readme and I received an error regarding this function which is private.

File: WAMessage/WAMessage.d.ts [hunk 1]
```diff
@@ -9548,7 +9548,7 @@ export namespace proto {
          * Converts this ContactMessage to JSON.
          * @returns JSON object
          */
-        public toJSON(): { [k: string]: any };
+        public static toJSON(): { [k: string]: any };
```
Comment (ibnusyawall) [inline WAMessage/WAMessage.d.ts:9551]: public static? Which one is true?

Comment (YehudaEi) [reply WAMessage/WAMessage.d.ts:9551]: My mistake - I meant it should be public static (and not an instance method).

**Comment by YehudaEi** [comment]
Hi @adiwajshing,
Did you understand my problem?

**Comment by adiwajshing** [comment]
Not really @YehudaEi

**Comment by YehudaEi** [comment]
What I was trying to do is run the example of sending a contact brought in readme.
I did nothing beyond that, and then I got an error (screenshot attached).
The way I solved the error was with adding a static before the function.

image
image

**Comment by YehudaEi** [comment]
Hi @adiwajshing,
Did you understand my problem now?

**Comment by YehudaEi** [comment]
Hi @adiwajshing,
Did you understand my problem now?

**Pull request closed without merging**